langserve/server.py

```diff
@@ -614,12 +614,8 @@ async def batch(
             ),
         )
 
-    @app.post(
-        namespace + "/c/{config_hash}/stream",
-        tags=route_tags_with_config,
```

Comment:
What is the before/after effect of this change?

Reply:
Before there are warnings for duplicate operation ids when launching the server, after the warnings are resolved.

The names and tags are used for documentation purposes for openapi spec. We've separated the documentation and implementation for most of the endpoints to be able to implement more complex validation of data. 

The impelmentation endpoints should not have any names/tags information since they're not surfaced in documentation